src/Components/Timer.tsx

```diff
@@ -10,13 +10,13 @@ const Timer = () => {
   console.log('Render timer') // 28 times
   const { gameState, gameDispatch } = useGame();
   const gameOver = useGameOverDocState();
-  const gamePaused = useGamePausedDocState();
-  const { weaponsStolen } = useWeaponsStolenDocState();
+  const sandTimerCount = useSandTimerState();
+  const weaponsStolen = useWeaponsStolenDocState();
```

Comment:
And this should be { weaponsStolen }

Reply:
I assume you got these changes after merging from main. But yea should be as my earlier comment